src/script/auth/component/LoginForm.tsx

```diff
@@ -92,40 +92,38 @@ const LoginForm = ({isFetching, onSubmit}: LoginFormProps) => {
         required
         data-uie-name="enter-email"
       />
-      <InputSubmitCombo>
-        <Input
-          name="password-login"
-          onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-            setPassword(event.target.value);
-            setValidPasswordInput(true);
-          }}
-          ref={passwordInput}
-          markInvalid={!validPasswordInput}
-          value={password}
-          autoComplete="section-login password"
-          type="password"
-          placeholder={_(loginStrings.passwordPlaceholder)}
-          pattern={'.{1,1024}'}
-          required
-          data-uie-name="enter-password"
-        />
-        {isFetching ? (
-          <Loading size={32} />
-        ) : (
-          <RoundIconButton
-            style={{marginLeft: 16}}
-            disabled={!email || !password}
-            type="submit"
-            formNoValidate
-            onClick={handleSubmit}
-            aria-label={_(loginStrings.headline)}
-            data-uie-name="do-sign-in"
-          >
-            <ArrowIcon />
-          </RoundIconButton>
-        )}
-      </InputSubmitCombo>
-    </InputBlock>
+      <Input
+        name="password-login"
+        onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+          setPassword(event.target.value);
+          setValidPasswordInput(true);
+        }}
+        ref={passwordInput}
+        markInvalid={!validPasswordInput}
+        value={password}
+        autoComplete="section-login password"
+        type="password"
+        placeholder={_(loginStrings.passwordPlaceholder)}
+        pattern={'.{1,1024}'}
+        required
+        data-uie-name="enter-password"
+      />
+      {isFetching ? (
```

Comment:
```suggestion
      />
      
      {isFetching ? (
```
Add spacing between lines